addons/stock/models/product.py

```diff
@@ -659,7 +659,7 @@ class ProductTemplate(models.Model):
 
     is_storable = fields.Boolean(
         'Track Inventory', store=True, compute='compute_is_storable', readonly=False,
-        default=False, precompute=True)
+        default=False, precompute=True, help='A storable product is a product for which you manage stock.\n')
```

Comment:
@dalaOdoo what would you expect as tooltip for the new `Track inventory` field ?